Update README.md

KWESIEMPRAH opened the pull request:
using variables for length and breadth

KWESIEMPRAH committed:
Update README.md

using variables for length and breadth

KWESIEMPRAH commented:
You would have have to create a sample before i can edit

trdhendup commented:
> You would have have to create a sample before i can edit

Yes, i created and wrote something about you. its in another repository. This one is last week's practice problem and no longer exists. please edit in the homework 3-part 1 repository. Thank you

KWESIEMPRAH commented:
No…
Read the question well.
If you don’t create the main branch I won’t be able to commit the last
required pull.


On Wed, 11 Oct 2023 at 8:44 PM, trdhendup ***@***.***> wrote:

> Closed #1 <https://github.com/trdhendup/Test/pull/1>.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/trdhendup/Test/pull/1#event-10625582711>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BC74HRJHI7E6O7ORWDLDSZLX644QPANCNFSM6AAAAAA5QDFHJI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

trdhendup commented:
> No… Read the question well. If you don’t create the main branch I won’t be able to commit the last required pull.
> […](#)
> On Wed, 11 Oct 2023 at 8:44 PM, trdhendup ***@***.***> wrote: Closed #1 <#1>. — Reply to this email directly, view it on GitHub <[#1 (comment)](https://github.com/trdhendup/Test/pull/1#event-10625582711)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BC74HRJHI7E6O7ORWDLDSZLX644QPANCNFSM6AAAAAA5QDFHJI> . You are receiving this because you authored the thread.Message ID: ***@***.***>

Sorry, Guess I added someone with similar name. Now i added through your profile and correct user name.

KWESIEMPRAH commented:
Ok, no problem.

I’ll check and update it

On Thu, 12 Oct 2023 at 8:55 PM, trdhendup ***@***.***> wrote:

> No… Read the question well. If you don’t create the main branch I won’t be
> able to commit the last required pull.
> … <#m_-7794915551322147481_>
> On Wed, 11 Oct 2023 at 8:44 PM, trdhendup *@*.*> wrote: Closed #1
> <https://github.com/trdhendup/Test/pull/1> <#1
> <https://github.com/trdhendup/Test/pull/1>>. — Reply to this email
> directly, view it on GitHub <#1 (comment)
> <https://github.com/trdhendup/Test/pull/1#event-10625582711>>, or
> unsubscribe
> https://github.com/notifications/unsubscribe-auth/BC74HRJHI7E6O7ORWDLDSZLX644QPANCNFSM6AAAAAA5QDFHJI
> <https://github.com/notifications/unsubscribe-auth/BC74HRJHI7E6O7ORWDLDSZLX644QPANCNFSM6AAAAAA5QDFHJI>
> . You are receiving this because you authored the thread.Message ID: @.*>
>
> Sorry, Guess I added someone with similar name. Now i added through your
> profile and correct user name.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/trdhendup/Test/pull/1#issuecomment-1760604186>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BC74HRIMWWGFAQZLKXLYXTLX7CGPVANCNFSM6AAAAAA5QDFHJI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

KWESIEMPRAH commented:
Did you update what we spoke on yesterday?

trdhendup commented:
Yes, I pulled and merged. Thanks a lot

KWESIEMPRAH commented:
You’re welcome 👍🏽

On Sat, 14 Oct 2023 at 1:06 AM, trdhendup ***@***.***> wrote:

> Yes, I pulled and merged. Thanks a lot
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/trdhendup/Test/pull/1#issuecomment-1762598887>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BC74HRMV72QKJTLZHGNAJITX7IMWTANCNFSM6AAAAAA5QDFHJI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
